data/brands/shop/shoes.json

```
@@ -10,6 +10,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "NOVO",
+      "locationSet": {"include": ["au"]},
```

```suggestion
      "locationSet": {"include": ["au", "nz"]},
```

Looks like they have 12 locations in NZ as well as 102 in AU.

https://www.novoshoes.co.nz/store-locator/

https://www.novoshoes.com.au/store-locator/